languages/nix.toml

```
@@ -0,0 +1,26 @@
+name = "nix"
+entrypoint = "main.sh"
+extensions = [
+  "sh"
+]
+packages = [
+]
+setup = [
+  """
+  mkdir -m 0755 /nix && chown runner /nix
+  su runner
+  curl -L https://nixos.org/nix/install | sh
+  nix-channel --update
+  nix search -u > /dev/null
```

```suggestion
  su runner -c 'curl -L https://nixos.org/nix/install | sh && nix-channel --update && nix search -u > /dev/null'
```

these commands get run one after another with no special consideration about what they can do (e.g. [this](https://github.com/replit/polygott/blob/de7ff18014a49deb9bb176036d60ad2d87379e4e/out/share/polygott/phase2.d/python3#L13-L21) is generatedy by [this](https://github.com/replit/polygott/blob/de7ff18014a49deb9bb176036d60ad2d87379e4e/languages/python3.toml#L21-L32)). by running the `su` command with `-c`, by the end of that command, the user substitution is only done for that one user.